include/fmt/format.h

```diff
@@ -3950,6 +3950,26 @@ class formatter<std::basic_string<Char, Traits, Allocator>, Char>
 template <typename Char, size_t N>
 struct formatter<Char[N], Char> : formatter<basic_string_view<Char>, Char> {};
 
+#if FMT_USE_BITINT
+namespace detail {
+template <class T> struct bit_int_format_as {
+  using traits = bitint_traits<T>;
+  static constexpr size_t N = traits::number_of_bits;
+  static constexpr bool is_signed = traits::is_signed;
+  using type = conditional_t<
+      N <= 64, conditional_t<is_signed, int64_t, uint64_t>,
+      conditional_t<(N > 64 && N <= 128),
+                    conditional_t<is_signed, __int128, unsigned __int128>,
+                    void>>;
+};
+}  // namespace detail
+
+template <typename T, typename Char>
+struct formatter<T, Char,
+                 enable_if_t<(detail::bitint_traits<T>::is_formattable)>>
+    : formatter<typename detail::bit_int_format_as<T>::type, Char> {};
```

Comment:
We could probably use `format_as` instead of specialization.

Reply:
> We could probably use `format_as` instead of specialization.

https://github.com/fmtlib/fmt/blob/a3f3f2ec9a2657acf0b5a762dc3979ac4900b9d7/include/fmt/base.h#L1420-L1428

Unless I'm misunderstanding what you mean, `format_as` will only work on `enum` or `class` types, of which `_BitInt` is neither?

Reply:
You are absolutely right.